src/code/PublishPSResource.cs

```diff
@@ -540,10 +540,37 @@ private bool IsValidModuleManifest(string moduleManifestPath)
                 {
                     if (privateData["PSData"] is Hashtable psData)
                     {
-                        if (psData.ContainsKey("Prerelease") && psData["Prerelease"] is string preReleaseVersion)
+                        if (psData.ContainsKey("prerelease") && psData["prerelease"] is string preReleaseVersion)
                         {
                             version = string.Format(@"{0}-{1}", version, preReleaseVersion);
                         }
+
+                        if (psData.ContainsKey("licenseUri") && psData["licenseuri"] is string licenseUri)

```

Comment:
typo in key name check
```suggestion
                        if (psData.ContainsKey("licenseuri") && psData["licenseuri"] is string licenseUri)

```